Specifications/catalog/catalog.protocol.md

```diff
@@ -0,0 +1,131 @@
+
+#### 3.3.1 ids:dataServiceType
+
+If the DataService refers to an IDS service endpoint, it must include the property `ids:dataServiceType`:
+
+| Category   | Description                                                                |
+|------------|----------------------------------------------------------------------------|
+| Definition | Specifies the IDS service type                                             |
+| Domain     | [dcat:DataService](https://www.w3.org/TR/vocab-dcat-2/#Class:Data_Service) |
+| Type       | xsd:string                                                                 |
+| Note       | The value of this field is left intentionally open for future extension.   |
+
+The following table lists well-know IDS endpoint types:
+
+| Value         | Description          |
+|---------------|----------------------|
+| ids:connector | A Connector endpoint |
+|               |                      |
+
+
+#### 3.3.2 dcat:servesDataset
+
+Note that the property `dcat:servesDataset` should be omitted from the `DataService` since `DataSets` are included as top-level entries. Clients are not required to process the
+contents of `dcat:servesDataset`.
+
+## 4 Technical Considerations
+
+### 4.1 Queries and Filter Expressions
+
+A _**catalog service**_ may support catalog queries or filter expressions as an implementation-specific feature. However, it is expected that query capabilities will be implemented
+by the consumer against the results of a `CatalogRequestMessage,` as the latter is an RDF vocabulary. Client-side querying can be scaled by periodically crawling provider catalog
+services, caching the results, and executing queries against the locally-stored catalogs.
+
+### 4.2 Replication Protocol
+
+The catalog protocol is designed to be used by federated services without the need for a replication protocol. Each consumer is responsible for issuing requests
+to 1..N catalog services, and managing the results. It follows that a specific replication protocol is not needed, or more precisely, each consumer replicates data from catalog
+services by issuing `CatalogRequestMessages`.
+
+The discovery protocol adopted by a particular dataspace defines how a consumer discovers catalog services.
+
+### 4.3 Security
+
+It is expected (although not required) that catalog services implement access control. A catalog as well as individual catalog _datasets_ may be restricted to trusted parties.
+The catalog service may require consumers to include a security token along with a `CatalogRequestMessage.` The specifics of how this is done can be found in the relevant
+catalog binding specification. In addition, this specification does not define the contents of the security token. It is expected different security mechanisms may be used such
+as [OAuth2 2.1](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-06) or [did:web](https://w3c-ccg.github.io/did-method-web/).
+
+### 4.4 Catalog Brokers
```

Comment:
That's a very interesting idea! Let's talk about it in one of the Thursday sessions.